textx/scoping/__init__.py

```diff
@@ -93,7 +93,7 @@ def __init__(self):
         self.filename_to_model = {}
 
     def has_model(self, filename):
-        return filename in self.filename_to_model.keys()
+        return filename in self.filename_to_model
```

Comment:
Is this the same or not (old code/new code)?

Reply:
don't understand the question. dict uses keys when iterating over it or with `in` operator so no need to explicitly call `keys()`.